utilities/load_testing/k6/utils.js

```diff
@@ -20,7 +20,7 @@ export const getProvider = (media_type) => {
 }
 
 export const REQUEST_HEADERS = {
-  Authorization: `Bearer ${__ENV.API_TOKEN}`,
+  Authorization: `Bearer ${__ENV.ACCESS_TOKEN}`,
```

Comment:
FWIW, We technically don't neven need to mess with this right now because rate limiting isn't working at all in production.

Reply:
When I tried this locally and let this run for a minute, I did receive a bunch of 429s:

```
     count_200......................: 28     0.421217/s
     count_429......................: 54     0.812347/s
     count_4xx......................: 21     0.315913/s
     data_received..................: 516 kB 7.8 kB/s
     data_sent......................: 10 kB  151 B/s
     http_req_blocked...............: avg=3.05ms   min=140ns    med=531ns    max=88.53ms  p(90)=927ns    p(95)=1.1µs   
     http_req_connecting............: avg=398.75µs min=0s       med=0s       max=11.46ms  p(90)=0s       p(95)=0s      
     http_req_duration..............: avg=217.89ms min=114.61ms med=172.89ms max=1.31s    p(90)=396.93ms p(95)=474.3ms 
       { expected_response:true }...: avg=266.33ms min=114.61ms med=181.57ms max=1.31s    p(90)=474.18ms p(95)=528.5ms 
     http_req_failed................: 52.44% ✓ 75       ✗ 68 
     http_req_receiving.............: avg=218.5µs  min=66.38µs  med=94.97µs  max=8.17ms   p(90)=228.44µs p(95)=268.77µs
     http_req_sending...............: avg=152.18µs min=32.06µs  med=176.75µs max=223.42µs p(90)=203.76µs p(95)=206.56µs
     http_req_tls_handshaking.......: avg=644.23µs min=0s       med=0s       max=19.35ms  p(90)=0s       p(95)=0s      
     http_req_waiting...............: avg=217.52ms min=114.38ms med=172.61ms max=1.31s    p(90)=396.7ms  p(95)=474.06ms
     http_reqs......................: 143    2.151216/s
     iteration_duration.............: avg=3.3s     min=3.11s    med=3.26s    max=4.4s     p(90)=3.49s    p(95)=3.61s   
     iterations.....................: 98     1.47426/s
     vus............................: 5      min=5      max=5
     vus_max........................: 5      min=5      max=5
```

So it seems that _some_ rate limiting is currently functioning.